app/src/main/cpp/skyline/services/pl/IPlatformServiceManager.cpp

```diff
@@ -30,4 +34,33 @@ namespace skyline::service::pl {
         response.copyHandles.push_back(handle);
         return {};
     }
+
+    Result IPlatformServiceManager::GetSharedFontInOrderOfPriority(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        auto languageCode{request.Pop<u64>()};
+
+        std::vector<u32> fontCodes;
+        std::vector<u32> fontOffsets;
+        std::vector<u32> fontSizes;
+
+        // TODO: actual font priority
+        for (std::size_t i = 0; i < core.fonts.size(); i++) {
+            fontCodes.push_back(static_cast<u32>(i));
+            auto region = core.fonts.at(i);
+            fontOffsets.push_back(region.offset);
+            fontSizes.push_back(region.length);
+        }
+
+        std::memset(request.outputBuf.at(0).data(), 0, request.outputBuf.at(0).size());
+        std::memset(request.outputBuf.at(1).data(), 0, request.outputBuf.at(1).size());
+        std::memset(request.outputBuf.at(2).data(), 0, request.outputBuf.at(2).size());
+
+        request.outputBuf.at(0).copy_from(fontCodes);
```

Comment:
Do you need to subspan them to min size? Maybe it's fine

Reply:
hmm, thats what yuzu had, but it depends if the outputbufs have some data beforehand